docs/concepts/executor/hub/debug-executor.md

```diff
@@ -1,59 +0,0 @@
-(debug-executor)=
```

Comment:
why are we removing this page?

Reply:
https://jinaai.slack.com/archives/C018F60RBL5/p1675679803276119

Reply:
Definitely happy to keep these, but need to re-add them to the index

Reply:
And they'll need a bit more proofing

Reply:
okay I restored them, you can proof read if you want